.github/workflows/lint.yml

```
@@ -44,6 +44,17 @@ jobs:
       - name: Clippy
         run: make clippy
 
+  taplo:
```

People probably aren't familiar with `taplo` so having it be `lint::toml` might be more informative.

```suggestion
  toml:
```